usr/src/uts/common/sys/fs/zfs.h

```diff
@@ -153,6 +153,10 @@ typedef enum {
 	ZFS_PROP_FILESYSTEM_COUNT,
 	ZFS_PROP_SNAPSHOT_COUNT,
 	ZFS_PROP_REDUNDANT_METADATA,
+	ZFS_PROP_ENCRYPTION,
+	ZFS_PROP_SALT,
+	ZFS_PROP_KEYSOURCE,
+	ZFS_PROP_KEYSTATUS,
```

Comment:
I think that Linux likes to have things added to the end of the enum so that new userland doesn't confuse old kernel modules?